lib/ITip/Broker.php

```diff
@@ -855,7 +855,10 @@ protected function parseEventInfo(VCalendar $calendar = null) {
                 $sequence = $vevent->SEQUENCE->getValue();
             }
             if (isset($vevent->EXDATE)) {
-                $exdate = $vevent->EXDATE->getParts();
+                foreach ($vevent->select('EXDATE') as $val) {
+                    $exdate = array_merge($exdate, $val->getParts());
```

Comment:
As discussed via Slack yesterday you: there is no relation between PRIORITY and EXDATE. And the exdates get sorted (so we have earliest to latest in our variable) after merging values of all EXDATE-properties.

Did I miss something?